common/cmake/devices.cmake

```diff
@@ -1332,9 +1332,11 @@ function(ADD_FPGA_TARGET)
   #   [INPUT_XDC_FILES <input_xdc_files>]
   #   [INPUT_SDC_FILE <input_sdc_file>]
   #   [EXPLICIT_ADD_FILE_TARGET]
+  #   [AUTO_ADD_FILE_TARGET]
   #   [EMIT_CHECK_TESTS EQUIV_CHECK_SCRIPT <yosys to script verify two bitstreams gold and gate>]
   #   [NO_SYNTHESIS]
   #   [ASSERT_USAGE <usage_spec>]
+  #   [ASSERT_TIMING <timing_spec>]
```

Comment:
No documentation for `ASSERT_TIMING`?

Reply:
Added documentation for new `ASSERT_TIMING` option